src/templates/domain/useCases/create.ejs

```diff
@@ -18,7 +18,8 @@ const useCase = ({ <%= props.name.camelCase %>Repository }) => () =>
     //Step description and function
     'Check if the <%= props.name.pascalCase %> is valid': step(ctx => {
       ctx.<%= props.name.camelCase %> = <%= props.name.pascalCase %>.fromJSON(ctx.req)
-
+      ctx.<%= props.name.camelCase %>.id = Math.floor(Math.random() * 100000)
```

Comment:
I think postgres or sqlserver create the id into an incremental way automatically, can you check it please?